fabfile/README.md

```diff
@@ -0,0 +1,55 @@
+# Fabric
+
+[Fabric](https://www.fabfile.org) is a Python library for performing devops tasks. You can thing of it a bit like a
+makefile on steroids for Python. Its api abstracts away the clunky way you would run shell commands in Python, check
+return values and manage stdio. Tasks may be targetted at particular hosts or group of hosts.
+
+# Using Fabric
+
+    $ cd PyBitmessage
+    $ fab <task_name>
+
+For a list of available commands:
+
+    $ fab -l
+
+General fabric commandline help
+
+    $ fab -h
+
+Arguments can be given:
+
+    $ fab task1:arg1=arg1value,arg2=arg2value task2:option1
+
+Tasks target hosts. Hosts can be specified with -H, or roles can be defined and you can target groups of hosts with -R.
+Furthermore, you can use -- to run arbitrary shell commands rather than tasks:
+
+    $ fab -H localhost task1
+    $ fab -R webservers -- sudo /etc/httpd restart
+
+# Getting started
+
+ * Install Fabric, fabric-virtualenv, virtualenvwrapper system-wide (log in to a new terminal afterwards if you're installing
```

Comment:
Hmm, it's weird. I've been able to run `mkvirtualenv` only after `. virtualenvwrapper.sh`.

Reply:
Did you log into a new shell? The installation updates your .bashrc or .profile.

Reply:
I'm on stretch here with virtualenvwrapper==4.3.1. The problem you describe with sourcing virtualenvwrapper.sh sounds like you just needed to log into a new shell after installing.

Reply:
Not every distro does so. For example Gentoo (which I use) does not. Quote from `virtualenvwrapper` documentation:
```
First, some initialization steps. Most of this only needs to be done one time. You will want to add
the command to source /usr/local/bin/virtualenvwrapper.sh to your shell startup file, changing the
path to virtualenvwrapper.sh depending on where it was installed by pip or your package manager.
```
I think `README.md` should rather refer to `virtualenvwrapper` documentation instead of assuming some installer magic.

Reply:
OK, I'll link to the virtualenvwrapper docs. I couldn't recall which it modified and was too lazy to check.